### PR TITLE
Add emoji images and css to project

### PR DESCRIPTION
**Pull request by jayroh**
michigan_benefits

### CODE REVIEW

File: app/assets/stylesheets/atoms/_emoji.scss [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  height: 1em;
+  width: 1em;
+  content: '';
+  background-size: 100% auto;
+  position: relative;
+}
+
+.emoji--inline {
+  top: .2em;
+
```
Comment (jessieay) [inline app/assets/stylesheets/atoms/_emoji.scss]: surprised rubocop let you get away with this

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -19,6 +19,7 @@
 @import 'atoms/form-widths';
 @import 'atoms/card';
 @import 'atoms/step-anchor';
+@import 'atoms/emoji';
```
Comment (jessieay) [inline app/assets/stylesheets/application.scss]: alpha order?

File: app/assets/stylesheets/atoms/_emoji.scss (hunk 1 above)
Comment (jayroh) [inline app/assets/stylesheets/atoms/_emoji.scss]: Huh, me too!

File: app/assets/stylesheets/application.scss (hunk 2 above)
Comment (jayroh) [inline app/assets/stylesheets/application.scss]: Unfortunately css can get completely screwy if you re-order rules/styles so I'm hesitant to re-order things too too much. Everything above is *not* alpha so I don't know if I want to make that change and untangle that. That cool, @jessieay?

Comment (jessieay) [inline app/assets/stylesheets/application.scss]: works for me!

**Pull request merged**